cluster/centos/util.sh

```diff
@@ -316,10 +333,59 @@ function provision-node() {
 #   KUBE_TEMP
 function ensure-setup-dir() {
   kube-ssh "${1}" "mkdir -p ${KUBE_TEMP}; \
+                   mkdir -p ${KUBE_TEMP}/etcd-cert; \
                    sudo mkdir -p /opt/kubernetes/bin; \
                    sudo mkdir -p /opt/kubernetes/cfg"
 }
 
+# Generate certificates for etcd cluster
+#
+# Assumed vars:
+#   $1 (etcd member name)
+#   $2 (master ip)
+function ensure-etcd-cert() {
+  ensure-setup-cfssl
+
+  local etcd_name="$1"
+  local master_ip="$2"
+
+  local cert_dir="${ROOT}/etcd-cert"
+  pushd "${cert_dir}"
+
+  echo "Generate CA certificates ..."
+  ./cfssl gencert -initca ca-csr.json | ./cfssljson -bare ca -
+
+  echo "Generate client certificates..."
```

Comment:
Suppose a pod running in a k8s cluster(peer certs only), the pod still can access the etcd service, so I think it's necessary.